### PR TITLE
Document use of alternative official supported images.

### PR DESCRIPTION
**Pull request by bendory**
References:

- #677
- #676 
- #672 
- #670 
- #663 
- #665 
- #667 
- #638 
- #322

### CODE REVIEW

File: bazel/README.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Bazel
 
-## Deprecation Notice
+## Altnernative Official Image
```
Comment (leeonlee) [inline bazel/README.md:3]: typo

File: bazel/README.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
-This builder is deprecated and will be deleted in an upcoming release.
-The recent version of the image will remain available for use. As a
-replacement, please use one of the supported images at
+A supported `bazel` image, including multiple tagged versions,
```
Comment (leeonlee) [inline bazel/README.md:5]: WDYT about replacing 'A supported' with 'The official'?

File: bazel/bazel.sh [hunk 3]
```diff
@@ -1,20 +1,14 @@
...
-echo
-echo For details, visit
-echo http://gcr.io/cloud-marketplace-containers/google/bazel
-echo
-echo           \*\*\*\*\* DEPRECATION NOTICE \*\*\*\*\*
-echo
+echo '
+                   ***** NOTICE *****
+
+A supported `bazel` image, including multiple tagged versions,
```
Comment (leeonlee) [inline bazel/bazel.sh:6]: Same as above

File: curl/README.md [hunk 4]
```diff
@@ -1,21 +1,18 @@
 # curl
 
-## Deprecation Notice
+## Altnernative Official Image
```
Comment (leeonlee) [inline curl/README.md]: Typo

File: bazel/README.md (hunk 2 above)
Comment (bendory) [reply bazel/README.md:5]: Hmmm, IDK if it's "the" only one.

File: bazel/bazel.sh (hunk 3 above)
Comment (bendory) [reply bazel/bazel.sh:6]: Same reply -- IDK if it's "the" or if there are others.

**Pull request merged**